src/vs/workbench/parts/explorers/browser/treeExplorerActions.contribution.ts

```diff
@@ -0,0 +1,57 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
@octref not sure if you care about it but this file could move into the `common` layer since it does not seem to depend on anything browser related.